project_hr/models/project_task.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    employee_ids = fields.Many2many(
+        comodel_name="hr.employee",
+        string="Linked employees",
+        compute="_compute_employee_ids",
+        store=True,
+    )
+    hr_category_ids = fields.Many2many(
+        comodel_name="hr.employee.category",
+        string="Employee Categories",
+        domain="[('id', 'in', allowed_hr_category_ids)]",
+        help="Here you can select the employee category suitable to perform "
+        "this task, limiting the selectable users to be assigned to "
+        "those that belongs to that category.",
+    )
+    allowed_hr_category_ids = fields.Many2many(
+        comodel_name="hr.employee.category",
+        string="Allowed HR categories",
+        compute="_compute_allowed_hr_category_ids",
+        help="Technical field for computing allowed employee categories "
+        "according categories at project level.",
+    )
+    # This field could have been named allowed_user_ids but a field with
+    # that name already exists in the Odoo core 'project' module
+    allowed_assigned_user_ids = fields.Many2many(
+        comodel_name="res.users",
+        string="Allowed users",
+        compute="_compute_allowed_assigned_user_ids",
+        help="Technical field for computing allowed users according employee "
+        "category.",
+    )
+
+    @api.depends("user_ids", "company_id")
+    def _compute_employee_ids(self):
+        for task in self.filtered("user_ids"):
+            task.employee_ids = task.user_ids.employee_ids.filtered(
+                lambda x: x.company_id == task.company_id
+            )[:1]
```

Comment:
```suggestion
            )
```

I think you should remove this

Reply:
thanks :)